src/main.cpp

```diff
@@ -2320,18 +2327,24 @@ void vcRenderWindow(vcState *pProgramState)
       {
         ImGui::PopStyleVar();
 
+        int sceneExplorerSize = pProgramState->settings.presentation.sceneExplorerSize;
+        if (pProgramState->sceneExplorerCollapsed)
+        {
+          sceneExplorerSize = 0;
+        }
```

Comment:
Unnecessary braces

Reply:
Braces Removed. And I think this is a quick UI operation, so the collapsed state is not saved. I added a blank button which still needs an icon.
![20200302154811](https://user-images.githubusercontent.com/11733679/75649181-154a4b00-5c9e-11ea-9702-5ddb532813e6.png)